etna/pipeline/autoregressive_pipeline.py

```diff
@@ -20,7 +20,18 @@
 class AutoRegressivePipeline(
     ModelPipelinePredictMixin, ModelPipelineParamsToTuneMixin, SaveModelPipelineMixin, BasePipeline
 ):
-    """Pipeline that make regressive models autoregressive.
+    """
+    Pipeline that make regressive models autoregressive.
+    
+    Makes forecast in several iterations, on each of them applies transforms and
+    predict step values by using forecast method of model.
```

Comment:
Let's wrap step into double ```. It will look like this:
```
predict ``step`` values by using forecast method of model.
```

It will make `step` look like a variable, because it is a parameter of the `AutoRegressivePipeline`.